Content.Client/_Impstation/CosmicCult/UI/Monument/MonumentMenu.xaml.cs

```diff
@@ -0,0 +1,136 @@
+        CultProgressBar.ForegroundStyleBoxOverride = new StyleBoxFlat { BackgroundColor = new Color(91, 62, 124) };
+
+        SelectGlyphButton.StyleClasses.Add("ButtonColorPurpleAndCool");
+        RemoveGlyphButton.StyleClasses.Add("ButtonColorPurpleAndCool");
+
+        UpdateBar(state);
+        UpdateEntropy(state);
+        UpdateGlyphs();
+        UpdateInfluences();
+    }
+
+    // Update all the entropy fields
+    private void UpdateBar(MonumentBuiState state)
+    {
+        CultProgressBar.Value = state.PercentageComplete;
+        ProgressBarPercentage.Text = Loc.GetString("monument-interface-progress-bar", ("percentage", state.PercentageComplete.ToString("0")));
+    }
+
+    // Update all the entropy fields
+    private void UpdateEntropy(MonumentBuiState state)
+    {
+        AvailableEntropy.Text = Loc.GetString("monument-interface-entropy-value", ("infused", state.AvailableEntropy.ToString()));
+        EntropyUntilNextStage.Text = Loc.GetString("monument-interface-entropy-value", ("infused", state.EntropyUntilNextStage.ToString()));
+        CrewToConvertUntilNextStage.Text = state.CrewToConvertUntilNextStage.ToString();
+    }
+
+    // Update all the glyph buttons
+    private void UpdateGlyphs()
+    {
+        var glyphs = _glyphPrototypes.ToList();
+        glyphs.Sort((x, y) =>
+            string.Compare(x.Name, y.Name, StringComparison.CurrentCultureIgnoreCase));
+
+        GlyphContainer.RemoveAllChildren();
+        foreach (var glyph in glyphs)
+        {
+            var boxContainer = new BoxContainer();
+            var unlocked = _unlockedGlyphProtoIds.Contains(glyph.ID);
+            var button = new Button
+            {
+                HorizontalExpand = true,
+                StyleClasses = { StyleBase.ButtonSquare },
+                ToolTip = Loc.GetString(glyph.Name),
+                Group = _glyphButtonGroup,
+                Pressed = glyph.ID == _selectedGlyphProtoId,
+                Disabled = !unlocked,
+                Modulate = !unlocked ? Color.Gray : Color.White,
+            };
+            button.OnPressed +=  _ => _selectedGlyphProtoId = glyph.ID;
+            var glyphIcon = new TextureRect
+            {
+                Texture = _sprite.Frame0(glyph.Icon),
+                TextureScale = new Vector2(2f, 2f),
+                Stretch = TextureRect.StretchMode.KeepCentered,
+            };
+            boxContainer.AddChild(button);
+            button.AddChild(glyphIcon);
+            GlyphContainer.AddChild(boxContainer);
+        }
+    }
+
+    // Update all the influence thingies
+    private void UpdateInfluences()
+    {
+        var influences = _influencePrototypes
+            .OrderBy(influence => _unlockedGlyphProtoIds.Contains(influence.ID))
+            .ThenBy<InfluencePrototype, string>(influence => influence.Name, StringComparer.CurrentCultureIgnoreCase);
```

Comment:
```suggestion
            .ThenBy<InfluencePrototype, string>(influence => influence.Name);
```
remove the referece to StringComparer, for some reason that decided to start tripping the sandboxing now (: